thirdparty/CMakeLists.txt

```diff
@@ -192,8 +192,13 @@ endif()
 # nlohmann json
 #
 if(ENABLE_SAMPLES OR NGRAPH_PDPD_FRONTEND_ENABLE OR NGRAPH_ONNX_FRONTEND_ENABLE OR NGRAPH_TF_FRONTEND_ENABLE)
-    add_subdirectory(nlohmann_json EXCLUDE_FROM_ALL)
+    set(JSON_Install ON)
+    add_subdirectory(json/nlohmann_json EXCLUDE_FROM_ALL)
     openvino_developer_export_targets(COMPONENT openvino_common TARGETS nlohmann_json)
+
+    set(BUILD_SHARED_LIBS OFF)
+    add_subdirectory(json/nlohmann_json_schema_validator EXCLUDE_FROM_ALL)
```

Comment:
it's better to wrap these 2 lines with `function` like here https://github.com/openvinotoolkit/openvino/blob/master/thirdparty/CMakeLists.txt#L35-L38

It adds new scope and we can set `BUILD_SHARED_LIBS` to `OFF` inside the function, while outside of the function variable will have default value

Reply:
fixed